gooddata-java-model/src/main/java/com/gooddata/sdk/model/executeafm/afm/filter/MeasureValueFilter.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2019, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.model.executeafm.afm.filter;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.gooddata.sdk.model.executeafm.Qualifier;
+import com.gooddata.sdk.model.executeafm.UriObjQualifier;
+import com.gooddata.util.GoodDataToStringBuilder;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Represents measure value filter applied on an insight.
+ */
+@JsonRootName(MeasureValueFilter.NAME)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class MeasureValueFilter implements ExtendedFilter, CompatibilityFilter, Serializable {
+
+    public static final String NAME = "measureValueFilter";
+
+    private static final long serialVersionUID = -3038654904981929337L;
+
+    private final Qualifier measure;
+    private final MeasureValueFilterCondition condition;
+
+    /**
+     * Creates a new {@link MeasureValueFilter} instance.
+     *
+     * @param measure The qualifier of referenced measure.
+     * @param condition The condition applied to a sliced measure value.
+     */
+    @JsonCreator
+    public MeasureValueFilter(
+            @JsonProperty("measure") final Qualifier measure,
+            @JsonProperty("condition") final MeasureValueFilterCondition condition) {
+
+        this.measure = measure;
+        this.condition = condition;
+    }
+
+    /**
+     * Copy itself using given uri qualifier
+     *
+     * @param qualifier qualifier to use for the new filter
+     *
+     * @return self copy with given qualifier
+     */
+    public MeasureValueFilter withObjUriQualifier(final UriObjQualifier qualifier) {
```

Comment:
ObjUri vs. UriObj mismatch

Reply:
This is in sync with similar FilterItem instances (but this is not FilterItem subtype so it can be changed): https://github.com/gooddata/gooddata-java/blob/3.0.0-RC/gooddata-java-model/src/main/java/com/gooddata/sdk/model/executeafm/afm/AbsoluteDateFilter.java#L64

Reply:
done